ingestion/functions/retrieval/valid_scheduled_event.json

```diff
@@ -1,4 +1,8 @@
 {
     "env": "local",
-    "sourceId": "5f6863d78a4a5200342cd2da"
+    "sourceId": "5f6863d78a4a5200342cd2da",
+    "parsingDateRange": {
```

Comment:
that doesn't make sense in the retrieval event right?

Reply:
yeah no please ignore I read this before understanding that we weren't storing the parsing date range in the source as we do for the date filter, so it needs to be passed like that.